tests/zkevm/test_worst_bytecode.py

```diff
@@ -204,3 +204,100 @@ def test_worst_bytecode_single_opcode(
         ],
         exclude_full_post_state_in_output=True,
     )
+
+
+@pytest.mark.valid_from("Cancun")
```

Comment:
```suggestion
@pytest.mark.valid_from("Paris")
```

Should this be Paris to also include pre-EIP-3860 forks? Test also seems to imply this by swapping out PUSH0 for PUSH1 pre-Shanghai

Reply:
This was the original idea, but because in Paris there is also no metering for initcode the benchmark behaves very differently. We rather need the Osaka implementation to test it on.

Reply:
As a lateral comment, targeting the latest fork should be the primary goal. While many of these tests could be generalized further to cover previous forks, I would prefer not to overcomplexify them since we aren't interested in benchmarking proof generations for older forks. If the generalization to cover older forks doesn't make the test more complex, I guess it's fine. Just mentioning this for some extra context.